src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -96,12 +107,32 @@ export default class PaypalCommercePaymentStrategy implements PaymentStrategy {
 
         await this._paypalCommercePaymentProcessor.initialize(paramsScript, undefined, gatewayId);
 
+        const fundingKey = this._paypalCommerceFundingKeyResolver.resolve(methodId, gatewayId);
+
         this._paypalCommercePaymentProcessor.renderButtons(cartId, container, buttonParams, {
             onRenderButton,
-            fundingKey: this._paypalCommerceFundingKeyResolver.resolve(methodId, gatewayId),
+            fundingKey,
             paramsForProvider: {isCheckout: true},
         });
 
+        if (this._isAPM)  {
+            /**
+             * FALLBACK for old checkouts that didn't pass apmFieldsContainer id
```

Comment:
rather than adding this comment, I would recommend adding a unit test not passing `apmFieldsContainer` param, which would ensure the logic is not accidentally removed.

Reply:
There was unit test for this case :)